README.md

```diff
@@ -66,8 +64,6 @@ After the prompts, this will create an initializer: `config/initializers/solarwi
 
 ## Sinatra
 
-![Sinatra](https://docs.appoptics.com/_images/sinatra.png)
-
```

Comment:
Yep. 
Rule of life - never link to external images or someone might decide to change them to a picture of a... kitten.